backend/packages/Upgrade/src/api/services/ExperimentAssignmentService.ts

```diff
@@ -1266,11 +1355,11 @@ export class ExperimentAssignmentService {
       experiment,
       exclusionCode: EXCLUSION_CODE.PARTICIPANT_ON_EXCLUSION_LIST,
     };
-    if (globallyExcluded.user || globallyExcluded.group.length) {
+    if (globallyExcluded.isUserExcluded || globallyExcluded.isGroupExcluded) {
       // store Individual exclusion document for the Group Exclusion as well:
       const promiseArray = [];
       promiseArray.push(this.individualExclusionRepository.saveRawJson([excludeUserDoc]));
-      if (globallyExcluded.group.length) {
+      if (globallyExcluded.isGroupExcluded) {
```

Comment:
@VivekFitkariwala Here we need the boolean for groupExcluded to store group exclusion document.